### PR TITLE
Support Node.js v8 and Update document

### PR DESCRIPTION
**Pull request by ryysud**
- Add Node.js v8 to Travis CI
- Update engines of package.json based on CI result
- Update document based on CI result

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 language: node_js
 node_js:
 - 'node'
+- '8'
```
Comment (jhecking) [inline .travis.yml]: Until Node.js v9 is released (October) Node.js v8 is covered by the `node` entry already.

File: package.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
   "license": "Apache-2.0",
   "main": "lib/aerospike",
   "engines": {
-    "node": ">=0.12"
+    "node": ">= 4.x"
```
Comment (jhecking) [inline package.json]: I was planning to make this change as part of PR #179.

File: README.md [hunk 3]
```diff
@@ -9,7 +9,7 @@
 
 An Aerospike add-on module for Node.js.
 
-This module is compatible with Node.js v4.x (LTS), v6.x (LTS) and v7.x. It
+This module is compatible with Node.js v4.x (LTS), v6.x (LTS) and v8.x (LTS). It
```
Comment (jhecking) [inline README.md]: According to the [LTS schedule](https://github.com/nodejs/LTS), Node.js v8 doesn't actually become an active LTS version until October 2017.

File: README.md [hunk 4]
```diff
@@ -99,14 +99,14 @@ Details about the API are available in the [`docs`](docs) directory.
 <a name="Prerequisites"></a>
 ## Prerequisites
 
-The aerospike package supports Node.js v0.12.x, v4.2.x (LTS) and v5.x as well
+The aerospike package supports Node.js v4.x (LTS), v6.x (LTS) and v8.x (LTS) as well
```
Comment (jhecking) [inline README.md]: Thanks for pointing out the conflicting version information listed in this section! I had missed this.

**Comment by jhecking** [comment]
Merged the changes to the README back to master in 97235dca7b343aff701cf54cf2282a61003f4f13. Thanks! Will apply the other changes with Node.js v9 is released and/or PR #179 gets merged.

File: README.md (hunk 3 above)
Comment (ryysud) [inline README.md]: Execuse me, I misunderstood that it is already LTS 🙇

**Comment by ryysud** [comment]
I understand it. Thank you for the information.

**Comment by ryysud** [comment]
@jhecking 

Please follow up on this.

> Merged the changes to the README back to master in 97235dc. Thanks! Will apply the other changes when Node.js v9 is released and/or PR #179 gets merged.

**Comment by jhecking** [comment]
@ryysud, I believe all of the changes you proposed have already been incorporated. Can you please clarify what changes exactly you think are still pending?

**Pull request closed without merging**